TAO/TAO_IDL/driver/drv_preproc.cpp

```diff
@@ -489,9 +489,7 @@ DRV_sweep_dirs (const char *rel_path,
                   /// case the original path argument included
                   /// spaces. If it didn't, no harm done.
                   ACE_CString incl_arg ("-I ");
-                  incl_arg += '\"';
                   incl_arg += bname;
-                  incl_arg += '\"';
                   DRV_cpp_putarg (incl_arg.c_str ());
```

Comment:
This one should definitely be two separate calls to `DRV_cpp_putarg`, because it's `-I FILE`.

Reply:
Looking at `drv_args.cpp` around line 297, the existing code supports (or attempts to support) either `-I dir` or `-Idir`, though since the latter is documented in the usage message I tend to think of it as preferred.

Based on what we've learned I think it would make sense to either change this (and any other similar calls) to `-Idir` or two calls to `DRV_cpp_putarg`.

Reply:
Do we have a decision on which way we want to handle this? I'm a little concerned about how this would creep the scope of this PR.

Reply:
I think it's easier to determine correctness of the fix if each call to `DRV_cpp_putarg` (and thus each entry in `DRV_arglist`) is intended to be a single argument for the preprocessor's `argv`.  It could be a separate PR.